<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmpFilter1</class>
 <widget class="QWidget" name="AmpFilter1">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMPPF</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Mode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>frequencyï¼š</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Order:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dSBFreq">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>20.000000000000000</double>
   </property>
   <property name="maximum">
    <double>20000.000000000000000</double>
   </property>
   <property name="value">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LINK_RILEY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BUTTER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BESSEL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BLACKMAN</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>201</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>S_12dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S_24dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S_36dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S_48dB</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
